frontend/src/app/modules/projects/create-project-modal/create-project-modal.component.ts

```diff
@@ -0,0 +1,66 @@
+import { Component, EventEmitter, OnInit, Output } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { MatDialogRef } from '@angular/material/dialog';
+import { NotificationService } from '@core/services/notification.service';
+import { ProjectService } from '@core/services/project.service';
+
+import { EngineEnum } from '../../../models/projects/engine-enum';
+import { ProjectDto } from '../../../models/projects/project-dto';
+
+@Component({
+    selector: 'app-create-project-modal',
+    templateUrl: './create-project-modal.component.html',
+    styleUrls: ['./create-project-modal.component.sass'],
+})
+export class CreateProjectModalComponent implements OnInit {
+    @Output() projectCreated = new EventEmitter<ProjectDto>();
+
+    selectedEngine: EngineEnum = EngineEnum.SqlServer;
+
+    projectForm: FormGroup;
+
+    constructor(
+        private fb: FormBuilder,
+        private projectService: ProjectService,
+        public dialogRef: MatDialogRef<CreateProjectModalComponent>,
+        private notificationService: NotificationService,
```

Comment:
in C# code style we specify access modifiers for each class member, so let's do the same in TS: add access modifier (public, private...) for each member and sort them (public members go first)